nikola/nikola.py

```diff
@@ -1660,6 +1675,11 @@ def generic_rss_renderer(self, lang, title, link, description, timeline, output_
             language=lang
         )
 
+        if copyright_ is True:
```

Comment:
Just do `if copyright_:` here. Add a comment for future readers that the second `if` is checking if any copyright info is set.